angular/src/directives/control-value-accessors/radio-value-accessor.ts

```diff
@@ -20,7 +20,7 @@ export class RadioValueAccessorDirective extends ValueAccessor {
   }
 
   @HostListener('ionSelect', ['$event.target'])
-  _handleIonSelect(el: any): void {
+  _handleIonSelect(el: HTMLInputElement): void {
```

Comment:
Interestingly, both `HTMLIonRadioElement` and `HTMLElement` are missing the `checked` property, so the `handleChangeEvent` call raises a type error when using either of those. I think I'm going to revert this one to `any` for now and add a TODO to revisit, since I've spent enough time on this task as it is 😆